modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/in_progress_forms_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module V0
+    class InProgressFormsController < ApplicationController
+      def update
+        form = InProgressForm.where(form_id:, user_uuid: @current_user.uuid).first_or_initialize
+        form.update!(form_data: params[:form_data] || params[:formData], metadata: params[:metadata])
+
+        render json: form, key_transform: :unaltered
+      end
+
+      def show
+        render json: form_for_user&.data_and_metadata
+      end
+
+      def destroy
+        raise Common::Exceptions::RecordNotFound, form_id if form_for_user.blank?
+
+        form_for_user.destroy
+        render json: form_for_user, key_transform: :unaltered
```

Comment:
Do we want to render the entire `form_for_user` on a destroy? Why not return a 204?

Reply:
This logic is also copied from `V0::InProgressFormsController`. I didn't want to change this, as it may have unintended effects on in_progress forms functionality in the fronted.

Reply:
@athifw The client does not use the value. I can walk you through the frontend code if you like but in the case of the remove it'd just be checking the [SIP actions file](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms/save-in-progress/actions.js).

Reply:
@athifw @ddzz @gabezurita Regarding the response body for delete, currently the VA backend returns the form, while the VA frontend does not use it. This brings up the question of what our methodology is going to be for maintaining the fact that our IPF API conforms to its client's expectations. Thinking through this a bit more now.

Reply:
I believe the correct methodology for safeguarding against client changes that would result in a breakage if our API isn't updated in concert is to expand the frontend's [`save-in-progess` test suite](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms/tests/save-in-progress/actions.unit.spec.js) with a group of examples that faithfully mock the behavior of our API.

Reply:
> I believe the correct methodology for safeguarding against client changes that would result in breakage if our API isn't updated in concert is to expand the frontend's [save-in-progess test suite](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms/tests/save-in-progress/actions.unit.spec.js) with a group of examples that faithfully mock the behavior of our API.

Agreed!